crates/fuel-core/Cargo.toml

```diff
@@ -62,6 +62,7 @@ tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
 tokio-rayon = { workspace = true }
 tokio-stream = { workspace = true, features = ["sync"] }
 tokio-util = { workspace = true }
+tower = { version = "0.4", features = ["limit"] }
```

Comment:
maybe we want to stick to the same version as `tower-http` here? not a blocker

Reply:
Sure thing. I've bumped `tower-http` in https://github.com/FuelLabs/fuel-core/pull/2309/commits/83103c0834321de0c28d584ff5da873159a4c8a9